retroshare-gui/src/gui/Posted/PostedItem.cpp

```diff
@@ -578,3 +580,26 @@ void PostedItem::toggleNotes()
 	}
 
 }
+
+void PostedItem::viewPicture()
+{
+	if(mPost.mImage.mData == NULL) {
+		return;
+	}
+
+	QString timestamp = misc::timeRelativeToNow(mPost.mMeta.mPublishTs);
+	QPixmap pixmap;
+	GxsIdDetails::loadPixmapFromData(mPost.mImage.mData, mPost.mImage.mSize, pixmap,GxsIdDetails::ORIGINAL);
+	RsGxsId authorID = mPost.mMeta.mAuthorId;
+	
+ 	PhotoView *PView = new PhotoView(this);
+	
+	PView->setPixmap(pixmap);
+	PView->setTitle(messageName());
+	PView->setName(authorID);
+	PView->setTime(timestamp);
+
+	PView->show();
+
+	/* window will destroy itself! */
```

Comment:
>window will destroy itself!

It won't.
You have to add this somewhere:
setAttribute ( Qt::WA_DeleteOnClose, true );

Reply:
its done